### ISSUE TITLE
heroku ci not available when install via MacOS installer

### ISSUE THREAD

**Issue by phardaker** [heroku/heroku-ci#45]
I installed heroku cli via the MacOS installer, and heroku ci is not available.

Tried totally uninstalling and re-installing, but still not available.

**Comment by michrome** [comment]
@phardaker what version of the Heroku CLI are you running? (`$ heroku version`)

**Comment by phardaker** [comment]
`heroku --version`

> heroku-cli/5.6.24-dbb4f5d (darwin-amd64) go1.7.4

`heroku ci`

> ci is not a heroku command.

**Comment by michrome** [comment]
Try running `$ heroku update` as I believe CI is included in recent versions (that is, it's not a plugin). e.g.

````
~ $ heroku version
heroku-cli/5.11.4-acbab72 (darwin-amd64) go1.7.5
~ $ heroku ci
=== Showing latest test runs for the myapp pipeline
~ $ heroku plugins
heroku-api-plugin 1.1.2
heroku-container-registry 4.1.3
heroku-enterprise 1.0.4
heroku-fastly 1.0.2
heroku-pg-extras 1.0.11
````

**Comment by phardaker** [comment]
Very odd - I tried running heroku update in the first instance.

Just tried that again, and this time it is working?!

The only difference is that I uninstalled heroku cli, installed via homebrew, confirmed heroku ci then worked, uninstalled homebrew heroku, re-installed using MacOS installer - still didn't work, now just run heroku update and voila! It is working.

We'll confirm on another person's Mac, maybe I just broke mine.
